src/NUnitFramework/framework/Internal/ParamAttributeTypeConversions.cs

```diff
@@ -66,13 +66,15 @@ private static IEnumerable GetData(object[] data, Type targetType)
                     continue;
                 }
 
-#if !NETSTANDARD1_6
+#if NETSTANDARD1_6
+                if (arg.GetType().FullName == "System.DBNull")
```

Comment:
Doesn't this version work for all platforms? I'd choose that (slightly less efficient?) over the additional conditionals, but that's preference. 😄

Reply:
Sold. Efficiency < maintainability unless profiling proves otherwise.